pontoon/base/static/js/translate.js

```diff
@@ -982,17 +982,36 @@ var Pontoon = (function (my) {
       });
     },
 
-
     /*
      * Update progress indicator and value
      */
     updateProgress: function () {
+
+      /*
+       * Count the number of words translated
+       */
+      function wordProgress () {
+        var entities = $("#entitylist .entity .string-wrapper .translation-string");
```

Comment:
Okay on closer inspection, I guess I figured it out. `Pontoon.entities[x].translation[0].string` being `null` means no translations made, and a string means the translated words. :smile: